server/infrastructures/repo/repo.go

```diff
@@ -0,0 +1,45 @@
+package repo
+
+import (
+	"database/sql"
+)
+
+// Repository has repository record
```

Comment:
__[Golint]__ comment on exported type Repo should be of the form "Repo ..." (with optional leading article)

<sub>posted by Sider</sub>